src/buckets/components/createBucketForm/BucketOverlayForm.tsx

```diff
@@ -164,7 +164,7 @@ export default class BucketOverlayForm extends PureComponent<Props> {
             <Accordion.AccordionHeader>
               <span>Advanced Configuration (Optional)</span>
             </Accordion.AccordionHeader>
-            <Accordion.AccordionBodyItem>
+            <Accordion.AccordionBodyItem testID="advancedSection">
```

Comment:
our convention with test ids is to follow html ids and classes in a loose [BEM style,](http://getbem.com/naming/) rather than camelCasing